beakerx/beakerx/install.py

```diff
@@ -32,10 +33,14 @@ def _all_kernels():
         'beakerx', os.path.join('static', 'kernel'))
     return [kernel for kernel in kernels if kernel != 'base']
 
+def convert_path(path):
```

Comment:
this should not be needed, that is the main point of this PR.
the reason for converting backslash to forward slash is because the old code was not adding quotes as needed.
but now that we are using json.dumps, it should insert the quotes for us, so we should not need to have special case handling for windows.